internal/test/plugins/capacity_static.go

```diff
@@ -111,6 +111,24 @@ func (p *StaticCapacityPlugin) Scrape(ctx context.Context, _ core.CapacityPlugin
 	return result, serializedMetrics, nil
 }
 
+func (p *StaticCapacityPlugin) BuildServiceCapacityRequest(backchannel core.CapacityPluginBackchannel, allAZs []limes.AvailabilityZone) (*liquid.ServiceCapacityRequest, error) {
+	return &liquid.ServiceCapacityRequest{
+		AllAZs: []liquid.AvailabilityZone{"az-one", "az-two"},
+		DemandByResource: map[liquid.ResourceName]liquid.ResourceDemand{
+			"first/capacity": {
```

Comment:
`first/capacity` is a pair of service type and resource name, but only a resource name should be used as a key here.

This is running up against a difference between how the old CapacityPlugins operate vs. how LIQUID operates. Capacity plugins are not restricted to a single service and can report resources from arbitrary services (hence their `Scrape` returns a data structure that is keyed on both service type and resource name).

There is no good way to solve this right now, which gives us the moral justification to just use a kludge and announce some stuff that is unrelated to the capacity plugin's actual configuration:

```suggestion
			"capacity": {
```